chia/rpc/wallet_rpc_api.py

```diff
@@ -1409,7 +1413,8 @@ async def nft_transfer_nft(self, request):
             return dict(success=False, error="target_address parameter missing")
         nft_wallet: NFTWallet = self.service.wallet_state_manager.wallets[wallet_id]
         try:
-            nft_coin_info = nft_wallet.get_nft_coin_by_id(bytes32.from_hexstr(request["nft_coin_id"]))
+            nft_coin_id = decode_puzzle_hash(request["nft_coin_id"])
```

Comment:
Actually, can we revert this one? Since we're still showing the NFT coin ID as hex (which I think is fine), I think it makes sense to keep this as a hex string.